app/models/activities/base_activity_provider.rb

```
@@ -187,6 +187,7 @@ def event_params(event_data)
     params
   rescue StandardError => e
     Rails.logger.error "Failed to deduce event params for #{event_data.inspect}: #{e}"
+    {}
```

Was this a bug?

---

It was intentional, but I'm not sure if there is much sense in it. Normally the method returns a `Hash`, but in case of exception it logs it and returns `true` from `Rails.logger.error` which is unexpected by the only use in `fill_events`. So returning empty `Hash` prevents rescuing one exception to cause another. And I am not sure about it all, as this just hides the exception, and I don't know if there are maybe legacy cases for errors that should be hidden. So maybe the correct way is just to remove rescue and logging altogether.

---

Removed it